client/js/screens/Home/Home.js

```diff
@@ -1,8 +1,29 @@
 import React from 'react';
-import {Text} from 'react-native';
+import {Text, ScrollView, Button} from 'react-native';
+import {removeViewer} from '../../config/models';
+import PropTypes from 'prop-types';
 
-const Home = props => {
-  return <Text>Hello Home</Text>;
+const Home = ({navigation}) => {
+  return (
+    <ScrollView>
+      <Text>Hello Home</Text>
+      <Button
+        onPress={async e => {
+          try {
+            await removeViewer();
+            await navigation.navigate('AuthLoading');
+          } catch (error) {
+            console.log(error);
```

Comment:
you typically don't want console.logs in the master branch

Reply:
This will be merged into the develop branch. I will remove all the consoles when it reaches the production stage.